op-supervisor/supervisor/backend/cross/safe_update_test.go

```diff
@@ -335,7 +335,7 @@ func TestScopedCrossSafeUpdate(t *testing.T) {
 		// but only on the second call (which will be used by HazardSafeFrontierChecks)
 		csd.deps.chainIDFromIndexfn = func() (eth.ChainID, error) {
 			defer func() { count++ }()
-			if count == 0 {
+			if count < 2 {
```

Comment:
I feel like this would totally change the test, but then I see the *existing* comment mentioning the second call -- what's up with this change?